cddl/examples/comid-integrity-registers.diag

```diff
@@ -20,32 +20,34 @@
           / comid.layer / 3 : 1
         }
       },
-      / measurement-map / {
-        / comid.mval / 1 : {
-          / comid.integrity-registers / 14 : {
-            0: [
-              [
-                / hash-alg-id / 1, / sha256 /
-                / hash-value / h'44aa336af4cb14a879432e53dd6571c7fa9bccafb75f488259262d6ea3a4d91b'
+      [
+        / measurement-map / {
+          / comid.mval / 1 : {
+            / comid.integrity-registers / 14 : {
```

Comment:
I'm a little surprised we're keeping integrity-registers when mkey can express the ir name.

Reply:
@deeglaze We can discuss Integrity registers removal in a separate issue and a follow up PR!

I deliberately did not wanted to make things complicated all in one go!